experiments/inference.py

```diff
@@ -44,6 +46,48 @@ def evaluate(
     return trues, preds, losses
 
 
+def predict_image(
```

Comment:
Zastąpiłbym wnętrze obecnej metody `infer` zamiast tworzyć nową.

Reply:
To trochę inne metody. 

`infer` to mała prosta metoda. Jest wykorzystywana do treningu, liczenia lossu i predykcji.

`predict_image` jest bardziej rozbudowana, narzuca transformacje na dane tak aby odwzorować te z testSetu (możemy użyć img z zewnątrz naszego datasetu). Dodatkowo ta metoda zwraca index przewidzianej klasy i printuje top3 klasy. Wrzucenie tej logiki do infer przeciąży tą metodę, może też się to odbić na performance przy uczeniu bo ta metoda jest wywoływana bardzo dużo razy (przy każdym batchu).